apm-sniffer/apm-sdk-plugin/mysql-8.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v8/define/PreparedStatementNullSetterInstrumentation.java

```diff
@@ -24,14 +24,15 @@
 import org.apache.skywalking.apm.agent.core.plugin.match.ClassMatch;
 import org.apache.skywalking.apm.plugin.jdbc.JDBCPreparedStatementNullSetterInstanceMethodsInterceptPoint;
 
-import static org.apache.skywalking.apm.agent.core.plugin.match.NameMatch.byName;
+import static org.apache.skywalking.apm.agent.core.plugin.match.MultiClassNameMatch.byMultiClassMatch;
 
 /**
  * @author kezhenxu94
  */
 public class PreparedStatementNullSetterInstrumentation extends AbstractMysqlInstrumentation {
```

Comment:
Why not extend PreparedStatementInstrumentation? Just override `getInstanceMethodsInterceptPoints`. Should be much easier and safer? Agree.

Reply:
make sense, save me a lot of codes, thanks